packages/common/src/filter-conditions/collectionSearchFilterCondition.ts

```diff
@@ -7,7 +7,7 @@ import { testFilterCondition } from './filterUtilities';
  */
 export const executeCollectionSearchFilterCondition: FilterCondition = (options: FilterConditionOption) => {
   // multiple-select will always return text, so we should make our cell values text as well
-  const cellValue = (options.cellValue === undefined || options.cellValue === null) ? '' : `${options.cellValue}`;
+  const cellValue = (options.cellValue === undefined || options.cellValue === null) ? '' : ((options.operator === 'IN_COLLECTION' || options.operator === 'NOT_IN_COLLECTION') && Array.isArray(options.cellValue)) ? options.cellValue.map(value => `${value}`) : `${options.cellValue}`;
```

Comment:
I find this line is starting to be way too lengthy, could we split this into multiple lines to increase readability. Actually, I rewrote the entire function and the unit tests are still good, could you please use the code below, I find it more readable. Thanks

```ts
export const executeCollectionSearchFilterCondition: FilterCondition = (options: FilterConditionOption) => {
  // multiple-select will always return text, so we should make our cell values text as well
  const filterOperator = options.operator;
  let cellValue: string | string[];
  if (Array.isArray(options.cellValue) && (filterOperator === 'IN_COLLECTION' || filterOperator === 'NOT_IN_COLLECTION')) {
    cellValue = options.cellValue.map(value => `${value}`);
  } else {
    cellValue = (options.cellValue === undefined || options.cellValue === null) ? '' : `${options.cellValue}`;
  }

  return testFilterCondition(filterOperator || 'IN', cellValue, options.searchTerms || []);
};
```

Reply:
Haha, yes that line got quite long now. I'm happy to change it, I'll update it soon 👍